### PR TITLE
Turn off ContextOverlay for edit, pal

### PR DESCRIPTION
**Pull request by davidkelly**
Plus, parenting, etc... for the 'highlight' box

### CODE REVIEW

File: scripts/system/pal.js [hunk 1]
```diff
@@ -883,6 +885,7 @@ function shutdown() {
     if (onPalScreen) {
         tablet.gotoHomeScreen();
     }
+    ContextOverlay.enable = true;
```
Comment (zfox23) [inline scripts/system/pal.js]: `enable` -> `enabled`

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1811/HighFidelity-Beta-PR11012-3bf81034a3bcfb48a673d8f61f9121e40049cc72.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1811/HighFidelity-Beta-PR11012-3bf81034a3bcfb48a673d8f61f9121e40049cc72.exe

File: scripts/system/pal.js (hunk 1 above)
Comment (davidkelly) [inline scripts/system/pal.js]: doh!

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1818/HighFidelity-Beta-PR11012-f365ce8abd545e0aa9f52c3f1a2dc67e06802fab.exe

**Pull request merged**